common/src/IO/SystemPaths.cpp

```diff
@@ -41,10 +41,10 @@ namespace TrenchBroom {
             }
             
             Path resourceDirectory() {
-#if defined __APPLE__
-                return IO::Path(wxStandardPaths::Get().GetResourcesDir().ToStdString());
-#else
+#if defined _WIN32
                 return appDirectory() + Path("Resources");
```

Comment:
The appveyor Windows build no longer has a Resources directory; alongside Trenchbroom.exe you have `shaders` `images`, etc. directories. see: https://ci.appveyor.com/project/kduske/trenchbroom/build/2.0.0-appveyor-157/artifacts

So I think either this needs to be `return appDirectory();` or the CMake configuration needs to be updated to bring back the `Resources` directory in Windows builds